.ci/install_kubernetes.sh

```diff
@@ -46,6 +46,11 @@ EOF"
 	chronic sudo -E apt update
 	chronic sudo -E apt install --allow-downgrades -y kubelet="$kubernetes_version" kubeadm="$kubernetes_version" kubectl="$kubernetes_version"
 elif [ "$ID" == "centos" ] || [ "$ID" == "fedora" ]; then
+	# Get fedora version
+	if [ "$ID" == "fedora" ]; then
+        	fedora_version=$(grep VERSION_ID /etc/os-release | cut -d '=' -f2)
```

Comment:
seems like os-release is `source`d at the beginning of the script, so this is not needed, you can use `VERSION_ID` instead